clientapp/app/components/Reusables/ButtonOne.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+interface IButtonOneProps {
+  text: string
+}
```

Comment:
nit: instead of the folder being called `"Reusables"`, i feel like the name `"shared"` is conventionally more appropriate since more codebases use that for reusable code